agent/php_execute.c

```diff
@@ -476,12 +476,8 @@ typedef struct _nr_library_table_t {
 static nr_library_table_t libraries[] = {
     {"Doctrine 2", "doctrine/orm/query.php", nr_doctrine2_enable},
     {"Guzzle 3", "guzzle/http/client.php", nr_guzzle3_enable},
-    /*
-     * NOTE:  This file ClientInterface.php also exists in Guzzle 3 and 6.
-     *        This file also exists in Predis.
-     */
-    {"Guzzle 4-5", "clientinterface.php", nr_guzzle4_enable},
-    {"Guzzle 6", "handlerstack.php", nr_guzzle6_enable},
+    {"Guzzle 4-5", "hasemitterinterface.php", nr_guzzle4_enable},
+    {"Guzzle 6", "guzzle/src/functions_include.php", nr_guzzle6_enable},
```

Comment:
Should this be `{"Guzzle 6-7", "guzzle/src/functions_include.php", nr_guzzle6_enable},`
for now?

Reply:
I was debating on this and decided to leave the functionality of the agent unchanged. This string is used to create a metric name, and that can be used to query data. Because this PR does not add feature to differentiate between Guzzle 6 and Guzzle 7, `Guzzle 6-7` means the same thing as `Guzzle 6` and I don't want to break existing queries. I hope future work, based on #498 , will leave `Guzzle 6` metric but add `Guzzle 7` supportability metric to precisely measure usage of different versions of Guzzle.